cmd/kubeadm/app/apis/kubeadm/v1alpha1/defaults.go

```diff
@@ -61,7 +62,7 @@ func SetDefaults_MasterConfiguration(obj *MasterConfiguration) {
 	}
 
 	if obj.CertificatesDir == "" {
-		obj.CertificatesDir = DefaultCertificatesDir
+		obj.CertificatesDir = path.Join(kubeadmapi.GlobalEnvParams.KubernetesDir, kubeadmconstants.CertificatesDirName)
```

Comment:
We strive to phase out kubeadmapi.GlobalEnvParams.KubernetesDir and try to use it as little as possible, please revert this